<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabla de Multiplicar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_top">
      <item>
       <widget class="QLabel" name="label_izquierda">
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #2E86C1;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../Archivo/recursos.qrc">:/Logos/logouat2.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_derecha">
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #2E86C1;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../Archivo/recursos.qrc">:/Logos/logouat.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_titulo">
      <property name="styleSheet">
       <string notr="true">font-size: 18px; font-weight: bold; color: #2E86C1;</string>
      </property>
      <property name="text">
       <string>Tabla de Multiplicar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_numero">
      <property name="styleSheet">
       <string notr="true">background-color: #EAFAF1; color: #1E8449; padding: 5px; border: 1px solid #1E8449; border-radius: 5px;</string>
      </property>
      <property name="placeholderText">
       <string>Ingrese un n√∫mero</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_calcular">
      <property name="styleSheet">
       <string notr="true">background-color: #2E86C1; color: white; padding: 10px; border-radius: 5px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_resultado">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #EAFAF1; color: #1E8449; padding: 10px; border: 1px solid #1E8449; border-radius: 5px; font-size: 14px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivo/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
